localization_manager/src/localization_manager_node.cpp

```diff
@@ -132,15 +134,13 @@ namespace localization_manager
         }
         catch (const std::exception& e)
         {
-            RCLCPP_ERROR_STREAM(get_logger(), "Uncaught Exception in localization_manager. Exception: "<< e.what() );
+            RCLCPP_ERROR_STREAM(rclcpp::get_logger("localization.localization_manager"), "localization_manager Uncaught Exception in localization_manager. Exception: "<< e.what() );
```

Comment:
On all logs, there is extra preceding string "localization_manager " on all strings to print. e.g.
`"localization_manager Uncaught Exception in localization_manager. Exception: "`. I am assuming it auto replace issue?

Reply:
Nice catch, I had updated them when logging wasn't working properly to try and find them easier. Fixed in the latest commit.